src/popper/parser.py

```diff
@@ -35,6 +35,7 @@ class WorkflowParser(object):
                                 "matching-rule": "any",
                                 "mapping": {"regex;(.+)": {"type": "str"}},
                             },
+                            "options": {"type": "map", "allowempty": True},
```

Comment:
Preferred to keep the yml validation simple as it would be unwieldy to specify all allowable options for all container runtimes